data/json/items/armor/gambesons.json

```diff
@@ -0,0 +1,1898 @@
+[
+  {
+    "id": "gambeson",
+    "type": "ARMOR",
+    "name": { "str": "padded gambeson" },
+    "description": "A padded greatcoat assembled by quilting readily available fabric and stuffing the layers with waste fabric or horse hair that was intended as standalone armor, being too thick to comfortably fit a cuirass over.  Likely either a historical artifact or a low-quality reproduction.  Though inferior to a proper layered aketon or gambeson it still offers decent protection.",
+    "weight": "11850 g",
+    "volume": "7500 ml",
+    "price": 20000,
+    "price_postapoc": 1500,
+    "to_hit": -5,
+    "material": [ "cotton" ],
+    "symbol": "[",
+    "looks_like": "jacket_light",
+    "color": "light_gray",
+    "warmth": 50,
+    "flags": [ "VARSIZE", "STURDY", "NORMAL", "OUTER", "BLOCK_WHILE_WORN" ],
+    "armor": [
+      {
+        "material": [
+          { "type": "cotton_quilted", "covered_by_mat": 100, "thickness": 2.5 },
+          { "type": "cotton", "covered_by_mat": 95, "thickness": 2.5 },
+          { "type": "cotton", "covered_by_mat": 95, "thickness": 2.5 }
+        ],
+        "covers": [ "torso" ],
+        "coverage": 100,
+        "encumbrance": 20
+      },
+      {
+        "material": [
+          { "type": "cotton_quilted", "covered_by_mat": 100, "thickness": 2 },
+          { "type": "cotton", "covered_by_mat": 95, "thickness": 2 },
+          { "type": "cotton", "covered_by_mat": 95, "thickness": 2 }
+        ],
+        "covers": [ "arm_l", "arm_r" ],
+        "coverage": 100,
+        "encumbrance": 25
+      },
+      {
+        "material": [ { "type": "cotton_quilted", "covered_by_mat": 100, "thickness": 2.0 } ],
+        "covers": [ "leg_l", "leg_r" ],
+        "specifically_covers": [ "leg_draped_l", "leg_draped_r" ],
+        "coverage": 72,
+        "encumbrance": 8,
+        "layers": [ "BELTED" ],
+        "warmth": 20
```

Comment:
This was `"//warmth": 20` in the other file, so effectively commented out. Unclear whether this is leftover from a past feature or intended to support a new feature for this sort of item composition.

Reply:
Yeah, I misunderstood that being something that can work - I'm gonna need to revise this more thoroughly, apologies.

Reply:
Commented out in: https://github.com/CleverRaven/Cataclysm-DDA/pull/64922